src/zope/securitypolicy/metadirectives.py

```diff
@@ -40,6 +41,12 @@ class IGrantDirective(IGrantAllDirective):
         description=u"Specifies the Permission to be mapped.",
         required=False)
 
+    permissions = Tokens(
+        title=u"Permissions",
+        description=u"Specifies a list of permissions to be mapped.",
```

Comment:
Maybe this should explicitly say "a whitespace-separated list of permissions"?

I don't remember how ZCML is documented; are the directive docs generated from interfaces?  If yes, then I think it's important to say how multiple permissions are specified (otherwise people might be tempted to try comma-separation or `[list, literals]`, or just get frustrated about the lack of docs).